contracts/src/internal/HyperdriveCheckpoint.sol

```diff
@@ -315,6 +325,23 @@ abstract contract HyperdriveCheckpoint is
             lpSharePrice
         );
 
+        // Claim the checkpoint reward on behalf of the sender.
+        //
+        // NOTE: We do this in a low-level call and ignore the status to ensure
+        // that the checkpoint will be minted regardless of whether or not the
+        // call succeeds.
+        if (_checkpointRewarder != address(0)) {
+            bool isTrader = _isTrader; // avoid stack-too-deep
+            (bool _success, ) = _checkpointRewarder.call(
+                abi.encodeCall(
+                    IHyperdriveCheckpointRewarder.claimCheckpointReward,
+                    (msg.sender, checkpointTime, isTrader)
+                )
+            );
```

Comment:
`returndata` bombing is possible here to cause OOG in `hyperdrive`:
- https://github.com/spearbit-audits/delv-week-review/issues/2